Sources/Apodini/Semantic Model Builder/InterfaceExporter/ConnectionContext.swift

```diff
@@ -0,0 +1,70 @@
+//
+//  ConnectionContext.swift
+//  
+//
+//  Created by Max Obermeier on 31.12.20.
+//
+
+import Foundation
+import Fluent
```

Comment:
I think it is better to replace this import with SwiftNIO. You can probably also remove Foundation

Reply:
Seemingly some left overs. Good caught!

Reply:
Resolved via 6c2b9bf